libs/core/src/lib/wizard/wizard.component.ts

```diff
@@ -269,7 +280,9 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
                 if (step.status === CURRENT_STEP_STATUS) {
                     const child = <HTMLElement>this.wrapperContainer.nativeElement.children[index];
                     this.wrapperContainer.nativeElement.scrollTo({
-                        top: child.offsetTop - this._elRef.nativeElement.querySelector('.' + WIZARD_NAVIGATION_CLASS).clientHeight,
+                        top:
```

Comment:
Do you mind moving `this._elRef.nativeElement.querySelector('.' + WIZARD_NAVIGATION_CLASS)` to separate variable and:
``` 
this.wrapperContainer.nativeElement.scrollTo({ top: child.offsetTop - navigation.clientHeight, behavior: 'smooth' });
```